Update flags description for subctl deploy-broker cmd (#471)
submariner-io/submariner-website: src/content/operations/deployment/subctl/_index.en.md

| `--kubecontext` `<string>`            | kubeconfig context to use
| `--repository` `<string>`             | The repository from where the various Submariner images will be sourced (default "quay.io/submariner")
| `--version` `<string>`                | Image version

> | `--components <strings>`              | The components to be installed - any of service-discovery,connectivity,globalnet. The defaults are: `service-discovery`,`connectivity`

Reviewer: ```suggestion
| `--components <strings>`              | Comma-separated list of components to be installed - any of `service-discovery`,`connectivity`,`globalnet`. The default is: `service-discovery`,`connectivity`
```

> | `--globalnet`                         | Enable support for overlapping Cluster/Service CIDRs in connecting clusters (default disabled)

Reviewer: On a side note, should `globalnet`  be deprecated and removed from the docs as well?

Author: good point, I see it still part of the listed flags in the deploy-broker command, so need to remove it from there as will

Reviewer: yes - it would be good to clean it up in the code.

| `--kubecontext` `<string>`            | kubeconfig context to use
| `--repository` `<string>`             | The repository from where the various Submariner images will be sourced (default "quay.io/submariner")
| `--version` `<string>`                | Image version
| `--service-discovery`                 | Enable Multi Cluster Service Discovery (default true)

> | `--globalnet`                         | Enable support for overlapping Cluster/Service CIDRs in connecting clusters (default disabled)

Reviewer: As per https://github.com/submariner-io/submariner-operator/pull/1228, we decided not to deprecate this flag.

Reviewer 2: oops right ':D

> | `--components <strings>`              | Comma-separated list of components to be installed - any of `service-discovery`,`connectivity`,`globalnet`. The default is: `service-discovery`,`connectivity`

Reviewer: I noticed in the code it says "_The default is all components_" which isn't correct and doesn't match here. While you're in the code changing the `globalnet` flag, can you update this as well?

Author: sure

Reviewer: As pointed out on https://github.com/submariner-io/submariner-operator/pull/1228, specifying just `--components globalnet` disables `service-discovery` & `connectivity` so doesn't make much sense.

Reviewer: Treating `globalnet` as a component seems problematic as it isn't standalone. Since we have the flag, I think we should remove it as a component. @sridhargaddam @mangelajo WDYT?

Reviewer 3: I tend to agree with @tpantelis. It's like a sub-flag of connectivity. Just thinking out loud, what are your thoughts on "connectivity, connectivity-with-globalnet"?

Reviewer: yeah we could do somethings like that. I think for v0.9 we should leave it as a flag, since it's pre-existing, but remove the `globalnet` component since that was introduced in v0.9.
